### ISSUE TITLE
Pickup points are not returned, config errors

### ISSUE THREAD

**Issue by bitbager** [Setono/SyliusPickupPointPlugin#5]
After I installed the plugin with the recent changes I get 

```
(1/1) InvalidArgumentException
Invalid service "setono.sylius_pickup_point.provider.post_nord": the value of argument "setono_sylius_pickup_point_post_nord.api_key" of method "Setono\SyliusPickupPointPlugin\Provider\PostNordProvider::__construct()" must be null, an instance of Symfony\Component\DependencyInjection\Reference or an instance of Symfony\Component\DependencyInjection\Definition, string given.
```
error. If I remove the `setono.sylius_pickup_point.provider.post_nord` service, the problem disapears. However, after I go through the checkout and select a shipping method with a GLS provider, I get no pickup points due to the Soap connection error.

**Comment by loevgaard** [comment]
Can you confirm it has been fixed, @bitbager ?

**Comment by bitbager** [comment]
I will check it later today. Is the SOAP connection fixed as well?

**Comment by loevgaard** [comment]
I didn't know the SOAP connection didn't work... It works when I test it

**Comment by bitbager** [comment]
As I wrote, I get no pickup points due to the Soap error from GLS provider. Here's a screenshot from my debug window. I work on a 100% clean setup. I added the GLS provider to the shipping method and proceed through the checkout selecting this shipping method.
<img width="1406" alt="screenshot 2019-02-28 at 10 21 31" src="https://user-images.githubusercontent.com/28542010/53555811-f8d37a00-3b42-11e9-96c8-edcf7d5f4736.png">
<img width="1427" alt="screenshot 2019-02-28 at 10 23 34" src="https://user-images.githubusercontent.com/28542010/53555814-f8d37a00-3b42-11e9-9da4-c78282a0b446.png">
